src/cpr_data_access/models.py

```diff
@@ -562,12 +621,24 @@ def text_block_after(self, text_block: TextBlock) -> Optional[TextBlock]:
 class CPRDocumentMetadata(BaseModel):
     """Metadata about a document in the CPR tool."""
 
-    publication_ts: Optional[datetime.datetime]
+    # NOTE: this is duplicated in the GST document metadata model intentionally,
```

Comment:
So to confirm the metadata is stored effectively in the input csv to the pipeline (output from retool). We are then updating the gst and base cpr document metadata objects to include more fields than before such that we can provide more filters etc.?

Reply:
yeah – this is just adding the fields that already exist in the metadata csvs [here](https://docs.google.com/spreadsheets/d/1-ixtkUWgJ1EjgP4cRE1WBeqgxfaxUk5c-QOg3vp7NTw/edit#gid=888413440) to each pydantic model